web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningList.jsx

```diff
@@ -83,7 +84,9 @@ const BackendCard = ({ backend, onStartTrain, onEdit, onDelete, onTestRequest })
               </Menu>
             }
           >
-            <Button type="link" icon={<IconEllipsis />} style={{ padding: "15px" }} />
+            <Button type="link" style={{ padding: "15px" }} aria-label="Machine learning model options">
```

Comment:
className would be ideal
```suggestion
            <Button type="link" className="p-15px" aria-label="Machine learning model options">
```